app/assets/javascripts/service_types/main.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import IsServiceWorking from '../service_types/IsServiceWorking.js';
+
+export default function renderIsServiceWorking() {
+  const serializedData = $('#serialized-data').data();
+
+  ReactDOM.render(
+    <IsServiceWorking
+      serializedData={serializedData}
```

Comment:
I think you can remove `serializedData` from this file?

Reply:
👍